Source/CarthageKit/Cartfile.swift

```diff
@@ -254,7 +255,33 @@ extension Dependency: Scannable {
 			}
 		} else if scanner.scanString("git", into: nil) {
 			parser = { urlString in
-				return .success(self.git(GitURL(urlString)))
+				
+				// This is a dependency expressed as git "..."
+				let gitURL = GitURL(urlString)
+				
+				if gitURL.containsGithubDomain {
```

Comment:
I have done so but make it fileprivate. Otherwise there would be globally accessible

```
Dependency(gitURL: GitURL) 
Dependency.git(GitURL)
```